src/app/clients/getExpandedClientDetails.ts

```diff
@@ -118,13 +123,13 @@ export const formatAddressFromClientDetails = (
 };
 
 export const formatHouseholdFromFamilyDetails = (
-    family: Pick<Schema["families"], "age" | "gender">[]
+    family: Pick<Schema["families"], "birth_year" | "gender">[]
 ): string => {
     let adultCount = 0;
     let childCount = 0;
 
     for (const familyMember of family) {
-        if (familyMember.age === null || familyMember.age >= 16) {
+        if (familyMember.birth_year <= getCurrentYear() - 17) {
```

Comment:
i think it might be worth pulling this logic of `determining whether a person is an adult` into a function as it's used in a few places with 16 / 17.